doc-surrealdb_versioned_docs/version-latest/surrealql/statements/define/access/index.mdx

```diff
@@ -67,6 +67,18 @@ DEFINE ACCESS account ON DATABASE TYPE RECORD
 ;
 ```
 
+## With `AUTHENTICATE` clause
+
+The authenticate clause can be used to change the record identifier returned by the `SIGNIN` and `SIGNUP` clauses or replace the identifier provided in the token when authenticating `WITH JWT` in the context of `DEFINE ACCESS ... TYPE RECORD`. 
+
+In the context of [`DEFINE ACCESS ... TYPE RECORD`](/docs/surrealdb/surrealql/statements/define/access/record), the `AUTHENTICATE` clause is always executed across signin, signup and token authentication. This clause can be used to log or stop authentication attempts from record users, as it is always executed across signin, signup and token authentication. The clause always expects a record identifier to be returned and will otherwise fail.
```

Comment:
The "is always executed across signin, signup and token authentication" sentence is written twice. The rest is correct :+1:

Reply:
I would maybe add: "The `THROW` statement can be called to return a custom error to the end user.".